gwpy/plot/tests/test_log.py

```diff
@@ -44,10 +47,27 @@ def formatter(cls):
             yield cls.TEST_CLASS()
 
     @pytest.mark.parametrize('x, fmt, result, texresult', [
-        pytest.param(0., None, r'$\mathdefault{0}$', '$0$', id="0"),
-        pytest.param(1, None, r'$\mathdefault{10^{0}}$', r'$10^{0}$',
-                     id="fmt=None"),
-        pytest.param(1, "%s", r'$\mathdefault{1}$', r'$1$', id="fmt=%s"),
+        pytest.param(
+            0.,
+            None,
+            r'$\mathdefault{0}$',
+            '$0$',
+            id="0",
+        ),
+        pytest.param(
+            1,
+            None,
+            r'$\mathdefault{10^{0}}$',
+            r'$\mathdefault{10^{0}}$' if mpl_version > "3.2.1" else "$10^{0}$",
```

Comment:
This line is the only changed functional line, everything else is just re-formatting.